README.md

```diff
@@ -57,89 +57,113 @@ export default {
 
 ## Options
 
-You can pass [eslint options](http://eslint.org/docs/developer-guide/nodejs-api#cliengine).
+You can pass [eslint options](https://eslint.org/docs/developer-guide/nodejs-api#%E2%97%86-new-eslint-options).
```

Comment:
The correct link seems to be: https://eslint.org/docs/developer-guide/nodejs-api#-new-eslintoptions

Reply:
Fixed. Thanks for review